avatars/lib/saferound.py

```diff
@@ -47,34 +79,47 @@ def saferound(iterable, places, strategy=DIFFERENCE, rounder=round, topline=None
             - strategy is not valid
             - values are not all floats
     """
-    assert isinstance(places, int)
-    assert strategy in [SMALLEST, LARGEST, DIFFERENCE]
```

Comment:
Are you sure an `Enum` is checked at runtime ?

Reply:
No, none of them are, but this is a lib that is used internally in our code, which we typecheck.
It though about keeping them in, but that would have resulted in many `if-else` checks, as `bandit` complains about asserts